paddle/framework/ddim.cc

```diff
@@ -195,6 +195,21 @@ std::vector<int64_t> vectorize(const DDim& ddim) {
   return result;
 }
 
+std::string debug_str(const DDim& ddim) {
```

Comment:
Is `debug_str` used?

Reply:
I use it a lot when debug the code

Reply:
removed since we can use cout << ddim to print the debug string.